fastlane_core/lib/fastlane_core/cert_checker.rb

```diff
@@ -139,30 +139,68 @@ def self.install_missing_wwdr_certificates
       # Install all Worldwide Developer Relations Intermediate Certificates listed here: https://www.apple.com/certificateauthority/
       missing = WWDRCA_CERTIFICATES.map { |c| c[:alias] } - installed_wwdr_certificates
       missing.each do |cert_alias|
-        install_wwdr_certificate(cert_alias)
+        Tempfile.create('fastlane-wwdr-cert-') do |tmpfile|
```

Comment:
```suggestion
        Tempfile.create(['fastlane-wwdr-cert-', '.cer']) do |tmpfile|
```

A minor suggestion here is to pass its file extension name with this array form so that you can still get a random file name with a valid file extension. It wouldn't matter here but it's just good practice.

https://ruby-doc.org/3.2.2/stdlibs/tempfile/Tempfile.html